hugo/layouts/partials/site/footer.html

```diff
@@ -1,4 +1,6 @@
 {{ $baseUrl := urls.Parse $.Site.Params.Baseurl }}
+{{ $issueUrl := print "https://github.com/cue-lang/docs-and-content/issues/new?template=issue_report.yml&cue-version=" .Permalink }}
```

Comment:
I /think/ you need to align the `template` param with the filename inside the other repo.

Compare the links underlying the "Get started" buttons on https://github.com/hashicorp/terraform/issues/new/choose and https://github.com/cue-lang/cue/issues/new/choose with the contents of their respective `.github/ISSUE_TEMPLATE/` directories ...